src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.test.tsx

```diff
@@ -48,7 +50,20 @@ const renderComponent = ({
   state?: Partial<BlastResultsState>;
 }) => {
   const initialState = {
-    blast: { blastResults: state ?? {} }
+    blast: {
+      blastResults: {
+        submissions: {},
+        ui: {
+          unviewedJobsPage: {
+            expandedSubmissionIds: []
+          },
+          viewedJobsPage: {
+            expandedSubmissionIds: []
+          }
+        },
```

Comment:
Is this initial object something worth importing as initial state from the slice?

Reply:
That's done